os/sys/log.c

```diff
@@ -160,6 +160,16 @@ log_lladdr_compact(const linkaddr_t *lladdr)
 }
 /*---------------------------------------------------------------------------*/
 void
+log_bytes(const void *data, size_t length)
+{
+  const uint8_t *u8data = (const uint8_t *)data;
+  size_t i;
+  for (i = 0; i != length; ++i) {
```

Comment:
Can you delete the space between for and the opening paren?

Reply:
Done.